generic/arch.h

```diff
@@ -271,6 +271,34 @@ struct Arch : BaseCtx
     bool place();
     bool route();
 
+    std::vector<IdString> getCellTypes() const {
+        return {};
```

Comment:
This should be being set up dynamically as bels are added to `generic`

Reply:
Fixed!